eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -748,7 +748,7 @@ object Helpers {
         val spendAnchors = htlcTxs.nonEmpty || onChainFeeConf.spendAnchorWithoutHtlcs
         val claimAnchorTxs: List[ClaimAnchorOutputTx] = if (spendAnchors) {
           // If we don't have pending HTLCs, we don't have funds at risk, so we can aim for a slower confirmation.
-          val confirmCommitBefore = htlcTxs.values.flatten.map(htlcTx => htlcTx.confirmBefore).minOption.getOrElse(currentBlockHeight + onChainFeeConf.feeTargets.commitmentWithoutHtlcsBlockTarget)
+          val confirmCommitBefore = htlcTxs.values.flatten.map(htlcTx => htlcTx.confirmBefore).minOption.getOrElse(currentBlockHeight + 1008) // TODO: setting a block target may not make sense when there is no urgency
```

Comment:
There is something incoherent with using a fixed target (even far in the future) for txs that are not urgent. In some full mempool scenarii we may find ourselves agressively bidding to get transactions confirmed when there is no real emergency.

Should we make the `confirmBefore` optional and reserve it to cases where there is a hard limit in terms of block height?

Reply:
> Should we make the `confirmBefore` optional and reserve it to cases where there is a hard limit in terms of block height?

I think that's a good idea indeed and probably what I should have done when I added the `commitmentWithoutHtlcsBlockTarget`. If we have no HTLC in the commitment, you're right that there is no specific block before which we must confirm, we should just use the priority (slow/medium/fast) that the node operator set for this case.

Reply:
Done in respectively 133264741 and 785d30ba8.